apps/console/webpack.config.ts

```diff
@@ -244,6 +277,40 @@ module.exports = (config: WebpackOptionsNormalized, context: NxWebpackContextInt
                 themeHash: getThemeConfigs(theme).styleSheetHash
             }) as unknown as WebpackPluginInstance
         );
+    } else if (process.env.PRE_AUTH_CHECK === "true") {
```

Comment:
Lets move this to a field level variable.

Suggestion: `isPreAuthCheckEnabled`